setup.py

```diff
@@ -8,7 +8,7 @@
 
 PKG_ROOT = os.path.dirname(__file__)
 
-VERSION = "0.2.20"
+VERSION = "0.1.71"
```

Comment:
This seems errornously committed.

Reply:
Your are right.. that is just valid for my_velux interaction